api/v1alpha1/openapi.yaml

```diff
@@ -393,7 +393,84 @@ paths:
       responses:
         "200":
           description: OK
-
+  /api/v1/sources/{id}/reports/{format}:
```

Comment:
format should be a query parameter.
In the path, the ids corresponds to resources like: sources, agents images